Kajabi/sage-lib
# fix(tooltip): DSSINBOX-4 resolve tooltip not removing

## Description

- [x] resolve bug where tooltip is not removing because mouseleave isn't getting fired on when the trigger is removed from the DOM

## Screenshots
**Vid of Tooltip fix**
https://github.com/Kajabi/sage-lib/assets/1241836/e2ad5348-50d4-42c7-8a8d-c6f10d3b3994

## Testing in `sage-lib`

- Visit the [Tooltip Rails page](http://localhost:4000/pages/component/tooltip?tab=preview)
- Verify that clicking on a tooltip trigger closes the tooltip content in addition to hovering

## Testing in `kajabi-products`

1. (**LOW**) Update `removeTooltip` to occur on Tooltip click .


## Related

[Closes DSSINBOX-4](https://kajabi.atlassian.net/browse/DSSINBOX-4)
